Bottom: 2d2187f2d75f5cb99350fffbbecea71a9196d9a7
Top:    d80073de79cd4fbceb4cf6e94a21d4f2ac7d7f66
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-30 19:47:49 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 4943755..353160a 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -285,6 +285,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
